interface-definitions/include/auth-local-users.xml.i

```diff
@@ -15,6 +19,71 @@
             <help>Password used for authentication</help>
           </properties>
         </leafNode>
+        <node name="otp">
+          <properties>
+            <help>2FA OTP authentication parameters</help>
+          </properties>
+          <children>
+            <leafNode name="key">
+              <properties>
+                <help>Token Key Secret key for the token algorithm (see RFC 4226)</help>
+                <valueHelp>
+                  <format>txt</format>
+                  <description>OTP key in hex-encoded format</description>
+                </valueHelp>
+                <constraint>
+                  <regex>[a-fA-F0-9]{20,10000}</regex>
+                </constraint>
+                <constraintErrorMessage>Key name must in hex be alphanumerical only (min. 20 hex characters)</constraintErrorMessage>
+              </properties>
+            </leafNode>
+            <leafNode name="otp-length">
+              <properties>
+                <help>Optional. Number of digits in OTP code (default: 6)</help>
```

Comment:
I don't think we should start help strings with "optional". Generally, I think most nodes should have no markings—if people miss something, they will see it in the commit error.

Reply:
Fixed